dojo/components/sql_group_concat.py

```diff
@@ -12,8 +12,8 @@ def __init__(
         super().__init__(
             expression,
             distinct="DISTINCT " if distinct else "",
-            ordering=" ORDER BY %s" % ordering if ordering is not None else "",
-            separator=' SEPARATOR "%s"' % separator,
```

Comment:
I never noticed this part of the code but this smells like a possible SQL-Injection. Are we sure about this implementation?

Reply:
Definitely not ideal... It looks like none of the fields in the init contain user supplied input, so I think it is safe as is now, but is a foot cannon in case it needs to be updated for any reason

Reply:
I looked as well @Maffooch and came to the same conclusion: No user data used in that code but is definitely on the list of things to refactor so it's not a problem in the future if someone changes things and user data makes it there.